_data/menu.yml

```diff
@@ -1,7 +1,7 @@
 
 options:
-  - title: Try Online
-    url: quickstart
+  - title: Playground
+    url: playground
```

Comment:
Could we still make a redirect for `[effekt-lang.org/quickstart](https://effekt-lang.org/quickstart.html)`? We use it a lot in external materials and I'd love to let the link still work...

Reply:
Also, we probably link to it on the main page and maybe even somewhere else.
https://github.com/search?q=repo%3Aeffekt-lang%2Feffekt-website%20%22quickstart%22&type=code

Reply:
oh yes, very good point